recipes/pcl/all/CMakeLists.txt

```
@@ -0,0 +1,4 @@
+cmake_minimum_required(VERSION 3.1.3)
```

```suggestion
cmake_minimum_required(VERSION 2.8.12)
```

don't need Cmake 3+ for this simple cmake file